src/Makefile

```diff
@@ -37,6 +37,8 @@ NVCCFLAGS += --compiler-options=-Wall
 LD = gcc
 LDFLAGS = -fPIC $(CUDA_LIB) -lcudart_static -lm -lstdc++
 
+CP = cp
```

Comment:
`install` is preferred over `cp` for installation. In particular, it can set the permissions of the destination file.